### PR TITLE
feat(capture-sdk):

### PR DESCRIPTION
**Pull request by AdnanYupi**
- only qr code scanning integration documentation

PIA-2977

[PIA-2977]: https://ginis.atlassian.net/browse/PIA-2977?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: capture-sdk/sdk/src/doc/source/features.rst [hunk 1]
```diff
@@ -139,6 +139,17 @@ To enable this feature simply pass ``true`` to ``GiniCapture.Builder.setQRCodeSc
     will not be ready at the time your app starts the camera screen and QR code detection will be silently disabled
     until the next time the camera screen starts.
 
+
+Only QR Code Scanning
+----------------
+
+The app supports QR code only scanning. To enable this feature simply pass ``true`` to ``GiniCapture.Builder.setOnlyQRCodeScanning``.
```
Comment (zladzeyka) [inline capture-sdk/sdk/src/doc/source/features.rst:146]:
```suggestion
The app supports QR code only scanning mode. To enable this feature simply pass ``true`` to ``GiniCapture.Builder.setOnlyQRCodeScanning``.
```

File: capture-sdk/sdk/src/doc/source/features.rst [hunk 2]
```diff
@@ -139,6 +139,17 @@ To enable this feature simply pass ``true`` to ``GiniCapture.Builder.setQRCodeSc
...
+When enabled this feature removes all UI elements related to taking picture from the Camera screen and leaves only QR code related UI elements.
```
Comment (zladzeyka) [inline capture-sdk/sdk/src/doc/source/features.rst:147]:
```suggestion
Enabling this feature removes all UI elements related to taking pictures from the Camera screen and leaves only QR code-related UI elements.
```

**Review by zladzeyka — CHANGES_REQUESTED** [review]
👍🏻 Good job.
Just 2 suggestions from my side.

**Review by zladzeyka — APPROVED** [review]
👏🏻🎉😊

**Review by llevente — APPROVED** [review]

**Pull request merged**